### PR TITLE
[agg_v2] e2e test with module publish

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs [hunk 1]
```diff
@@ -382,6 +382,28 @@ proptest! {
         );
     }
 
+    #[test]
+    fn test_aggregator_with_republish(test_env in arb_test_env(6), element_type in arb_agg_type(), use_type in arb_use_type()) {
```
Comment (gelash) [inline aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs:386]: Should somehow remember to adjust it (to failpoints) when we get rid of the fallback

**Review by gelash — APPROVED** [review]

File: aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs (hunk 1 above)
Comment (igor-aptos) [reply aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs:386]: we should probably just have both?

Comment (igor-aptos) [reply aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs:386]: as after your PR , different path will be executed

what is the faillpoint name you have?

Comment (gelash) [reply aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs:386]: good point

**Review by vusirikala — APPROVED** [review]

File: aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs (hunk 1 above)
Comment (gelash) [reply aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs:386]: fail-point-resource-group-serialization
https://github.com/aptos-labs/aptos-core/blob/e34cd18d6207c1422eaf6215638c93918b83445d/aptos-move/block-executor/src/executor.rs#L780C14-L780C53

it also enables in fallback though, but we can adjust it later.

**Pull request merged**